MaterialLibrary/src/main/java/io/doist/material/widget/utils/MaterialWidgetHandler.java

```diff
@@ -151,4 +195,40 @@ public static void init(View view, AttributeSet set, int defStyle, int[] attrs)
         }
         return newArray;
     }
+
+    private static Drawable getDrawable(MaterialResources r, TypedArray ta, int index) {
```

Comment:
I don't think `MaterialWidgetHandler` should handle any part of `getDrawable`. We should always use `MaterialResources`. More so after the merge with https://github.com/Doist/AndroidMaterial/commit/eafcefa0694ea20aafb9ababfcf9bf075c113bd3.
What do you think?